components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/mappings/APIMappingUtil.java

```diff
@@ -320,8 +320,8 @@ public static APIDTO fromAPItoDTO(APIProduct model, String tenantDomain) throws
         List<APITiersDTO> tiersToReturn = new ArrayList<>();
 
         int tenantId = 0;
-        if (!StringUtils.isBlank(tenantDomain)) {
-            tenantId = APIUtil.getTenantIdFromTenantDomain(tenantDomain);
+        if (!StringUtils.isBlank(organization)) {
+            tenantId = APIUtil.getTenantIdFromTenantDomain(organization);
```

Comment:
Same here on getting the tenant ID

Reply:
fixed in https://github.com/wso2/carbon-apimgt/pull/10553/commits/448f1494c616bef5b514c816f3a29c6dda74a465